src/content/deployment/flavors.md

```diff
@@ -52,32 +58,63 @@ Open your project in Xcode.
 </li>
 <li>
 
-Select **Product** > **Scheme** > **New Scheme** from the menu to
-add a new `Scheme`.
+Navigate to the **Build Settings** of the target(s)
+and follow these instructions:
+
+</li>
+
+If `DART_DEFINES` doesn't exist yet:
+
+* Click the **+** button in the top left.
+* Select **Add User-Defined**.
+* Enter `DART_DEFINES` as the name.
+* Enter the **base64 encoded version**
+  of `FLUTTER_APP_FLAVOR=<value>` as the value,
+  where `value` is the name of the flavor
+  that is expected to be the `appFlavor` in Dart.
```

Comment:
It looks like this is the first time we reference `appFlavor` in this doc, consider making this a link too:

```suggestion
  that is expected to be the [`appFlavor`][] in Dart.
```